src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.cxx

```diff
@@ -331,6 +331,82 @@ t8_geometry_triangulated_spherical_surface::t8_geom_evaluate (t8_cmesh_t cmesh,
   }
 }
 
+/**
+ * Map the faces of a unit cube to a spherical surface.
+ * \param [in]  cmesh      The cmesh in which the point lies.
+ * \param [in]  gtreeid    The global tree (of the cmesh) in which the reference point is.
+ * \param [in]  ref_coords  Array of \a dimension many entries, specifying a point in [0,1]^dimension.
+ * \param [out] out_coords  The mapped coordinates in physical space of \a ref_coords.
+ */
+void
+t8_geometry_quadrangulated_spherical_surface::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid,
+                                                                const double *ref_coords, const size_t num_coords,
+                                                                double *out_coords) const
+{
+  double n[3]; /* Normal vector. */
+  double r[3]; /* Radial vector. */
+  double p[3]; /* Vector on the plane. */
+
+  t8_locidx_t ltreeid = t8_cmesh_get_local_id (cmesh, gtreeid);
+  double *tree_vertices = t8_cmesh_get_tree_vertices (cmesh, ltreeid);
+
+  {
+    const double center_ref[3] = { 0.5, 0.5, 0.0 };
+    t8_geom_linear_interpolation (center_ref, tree_vertices, 3, 2, n);
+
+    /* Normalize vector `n`. */
+    const double norm = sqrt (n[0] * n[0] + n[1] * n[1] + n[2] * n[2]);
+    n[0] = n[0] / norm;
+    n[1] = n[1] / norm;
+    n[2] = n[2] / norm;
+  }
```

Comment:
Maybe this would be a good function in t8_vec.c

Reply:
Done.